cmd/process-agent/app/check.go

```diff
@@ -103,6 +103,11 @@ func runCheckCmd(cmd *cobra.Command, args []string) error {
 	log.Infof("running version: %s", agentVersion.GetNumberAndPre())
 
 	// Start workload metadata store before tagger (used for containerCollection)
+	if ddconfig.Datadog.GetBool("process_config.remote_workloadmeta") {
+		if err = workloadmeta.ConfigureAsRemote(); err != nil {
```

Comment:
does `.ConfigureAsRemote()` retries several time? 
I'm wondering what happen if the process-agent start before the `core-agent`

Reply:
The retries are handled when the remote workloadmeta collector starts: here: https://github.com/DataDog/datadog-agent/pull/14482/files#diff-a48cb39c4455a8d213283cd15194c52730ed1bce9e6413533ef597776d3234e3R109